### ISSUE TITLE
Focusable scrollable areas

### ISSUE THREAD

**Issue by gsproston-scottlogic** [gsproston-scottlogic/prompt-injection#509]
All areas that are scrollable must be focusable, and reachable by keyboard navigation only.
https://www.w3.org/WAI/standards-guidelines/act/rules/0ssw9k/

- [x] Find which areas can scroll 
- [x] Make them focusable

Note: 

- Firefox automatically makes scrollable areas focusable, so this issue won't be seen on that browser. Edge and Chrome definitely have this issue.
- Some components might only become scrollable on smaller screen sizes.
- This issue applies to vertical AND horizontal scrolling.

## AC
GIVEN an HTML element is scrollable
THEN it should also be focusable

GIVEN an HTML element is scrollable
THEN it should be reachable by keyboard navigation only

GIVEN an HTML element is scrollable
WHEN a user is focused on it
THEN the user should be able to use keyboard arrows to scroll
